core/shared/src/main/scala/interface.scala

```diff
@@ -46,6 +46,17 @@ trait Param[Typeclass[_], Type] {
   /** the name of the parameter */
   def label: String
 
+  /** flag indicating a repeated (aka. vararg) parameter
+    *
+    * For example, for a case class,
+    * <pre>
+    * case class Account(id: String, emails: String*)
+    * </pre>
+    * the [[Param]] instance corresponding to the `emails` parameter would be `repeated` and have a
+    * [[PType]] equal to the type `Seq[String]`. Note that only the last parameter of a case class
+    * can be repeated. */
+  def repeated: Boolean
```

Comment:
I'm happy for this additional parameter to be here. It's an extra parameter to the interface, but in #46 I'm going to offer a way to make that optional (which might result in a tiny runtime and compile-time speedup).